backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Graphics.java

```diff
@@ -73,23 +74,35 @@ public class Lwjgl3Graphics extends AbstractGraphics implements Disposable {
 
 		@Override
 		public void invoke (long windowHandle, final int width, final int height) {
-			if (posted) return;
-			posted = true;
-			Gdx.app.postRunnable(new Runnable() {
-				@Override
-				public void run () {
-					posted = false;
-					updateFramebufferInfo();
-					if (!window.isListenerInitialized()) {
-						return;
-					}
-					window.makeCurrent();
-					gl20.glViewport(0, 0, backBufferWidth, backBufferHeight);
-					window.getListener().resize(getWidth(), getHeight());
-					window.getListener().render();
-					GLFW.glfwSwapBuffers(windowHandle);
+			if (SharedLibraryLoader.isWindows || SharedLibraryLoader.isLinux) {
```

Comment:
I think we should replace this with `Configuration.GLFW_CHECK_THREAD0.get(true)` as @piotr-j suggested.